web/hand_tool.js

```diff
@@ -35,30 +35,48 @@ var GrabToPan = grabToPan.GrabToPan;
 var Preferences = preferences.Preferences;
 var SecondaryToolbar = secondaryToolbar.SecondaryToolbar;
 
-var HandTool = {
-  initialize: function handToolInitialize(options) {
-    var toggleHandTool = options.toggleHandTool;
+/**
+ * @typedef {Object} HandToolOptions
+ * @property {HTMLDivElement} container - The document container.
+ * @property {HTMLButtonElement} toggleHandTool - The button element for
+ *                                                toggling the hand tool.
+ */
+
+/**
+ * @class
+ */
+var HandTool = (function HandToolClosure() {
+  /**
+   * @constructs HandTool
+   * @param {HandToolOptions} options
+   */
+  function HandTool(options) {
+    this.container = options.container;
+    this.toggleHandTool = options.toggleHandTool;
+
     this.handTool = new GrabToPan({
-      element: options.container,
+      element: this.container,
       onActiveChanged: function(isActive) {
-        if (!toggleHandTool) {
+        if (!this.toggleHandTool) {
           return;
         }
         if (isActive) {
-          toggleHandTool.title =
+          this.toggleHandTool.title =
             mozL10n.get('hand_tool_disable.title', null, 'Disable hand tool');
-          toggleHandTool.firstElementChild.textContent =
+          this.toggleHandTool.firstElementChild.textContent =
             mozL10n.get('hand_tool_disable_label', null, 'Disable hand tool');
         } else {
-          toggleHandTool.title =
+          this.toggleHandTool.title =
             mozL10n.get('hand_tool_enable.title', null, 'Enable hand tool');
-          toggleHandTool.firstElementChild.textContent =
+          this.toggleHandTool.firstElementChild.textContent =
             mozL10n.get('hand_tool_enable_label', null, 'Enable hand tool');
         }
-      }
+      }.bind(this)
     });
-    if (toggleHandTool) {
-      toggleHandTool.addEventListener('click', this.toggle.bind(this), false);
+
+    if (this.toggleHandTool) {
+      this.toggleHandTool.addEventListener('click', this.toggle.bind(this),
+                                           false);
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters, `false` is the default here, so can't we just remove that?